eitprocessing/features/pixel_breath.py

```diff
@@ -0,0 +1,208 @@
+import itertools
+from collections.abc import Callable
+from dataclasses import dataclass, field
+
+import numpy as np
+
+from eitprocessing.datahandling.breath import Breath
+from eitprocessing.datahandling.continuousdata import ContinuousData
+from eitprocessing.datahandling.eitdata import EITData
+from eitprocessing.datahandling.intervaldata import IntervalData
+from eitprocessing.datahandling.sequence import Sequence
+from eitprocessing.features.breath_detection import BreathDetection
+
+
+@dataclass
+class PixelBreath:
+    """Algorithm for detecting timing of pixel breath and deflation in pixel impedance data.
```

Comment:
You probably replace "inflation" with breath everywhere, resulting is a weird sentence here. Please check the documentation.